pynetbox/core/query.py

```diff
@@ -185,9 +185,9 @@ def get_version(self):
         :Returns: Version number as a string. Empty string if version is not
         present in the headers.
         """
-        headers = {
-            "Content-Type": "application/json",
-        }
+        headers = {"Content-Type": "application/json"}
+        if self.token:
+            headers["authorization"] = "Token {}".format(self.token)
```

Comment:
I think that more natural way for requests is to use custom auth middleware https://requests.readthedocs.io/en/latest/user/advanced/#custom-authentication

Reply:
@doc-sheet Please open a new issue/PR if you want to change the authorization header logic, let's keep this PR about the major issue at hand.